### ISSUE TITLE
the problem with counting fantasy points

### ISSUE THREAD

**Issue by bubic228** [ValveSoftware/Dota2-Gameplay#12786]
### Description

in fantasy, I have yatoro, kiyotaka and miposhka, but the points were counted only for the first two. My steam id 1085312770

### Screenshots

![20231013172339_1](https://github.com/ValveSoftware/Dota2-Gameplay/assets/147839593/c7b4b9b0-03d1-4da7-a4ea-b6c5e2c2b4f2)

**Comment by aggoo** [comment]
I also have the same problem.
![Capture](https://github.com/ValveSoftware/Dota2-Gameplay/assets/38524176/9a93ac71-fe89-4fab-9297-9302eadfc620)

**Comment by xstrei06** [comment]
Me aswell, also suddenly all the top scoring people gained like 30k+ score, seems like something weird is happening
![image](https://github.com/ValveSoftware/Dota2-Gameplay/assets/101869862/a7922fc6-8268-4979-8038-0d04c197c1af)

**Comment by SoftPancakes42** [comment]
> Me aswell, also suddenly all the top scoring people gained like 30k+ score, seems like something weird is happening ![image](https://user-images.githubusercontent.com/101869862/274998983-a7922fc6-8268-4979-8038-0d04c197c1af.png)

Yeah I saw that - I'm experiencing the same thing. It's kind of frustrating because this morning I had 55k and then it dropped to 40k with the bug and now everyone else is somehow getting points and my stuff isn't updating? 

It really ruins the fun I want to have with the fantasy element.

**Comment by iciao1** [comment]
+++

**Comment by SoftPancakes42** [comment]
Adding to this - rerolls are bugged. I got rerolls from a new level and it didn't give them to me. Please fix.

**Comment by heykean123** [comment]
![275121622-50e70a98-4f4f-41dd-b4a7-73e5db35414e](https://github.com/ValveSoftware/Dota2-Gameplay/assets/147894401/1f0c1461-6cb0-4e65-8d30-aecc3a5ceddc)
i have the same problem please help fix day 2 not counting then day 1 i got deducted my steam id is: 163199572
